os/os.module.go

```diff
@@ -0,0 +1,284 @@
+	}
+	return py.Bytes([]byte(dir)), nil
+}
+
+// chdir changes the current working directory to the provided path.
+func chdir(self py.Object, args py.Tuple) (py.Object, error) {
+	if len(args) == 0 {
+		return nil, py.ExceptionNewf(py.TypeError, "Missing required argument 'path' (pos 1)")
+	}
+	dir, ok := args[0].(py.String)
+	if !ok {
+		return nil, py.ExceptionNewf(py.TypeError, "str expected, not "+args[0].Type().Name)
+	}
+	err := os.Chdir(string(dir))
+	if err != nil {
+		return nil, py.ExceptionNewf(py.NotADirectoryError, "Couldn't change cwd; "+err.Error())
+	}
+	return py.None, nil
+}
+
+// getenv returns the value of the environment variable key.
+// If no such environment variable exists and a default value was provided, that value is returned.
+func getenv(self py.Object, args py.Tuple) (py.Object, error) {
+	if len(args) < 1 {
+		return nil, py.ExceptionNewf(py.TypeError, "missing one required argument: 'name:str'")
+	}
+	k, ok := args[0].(py.String)
+	if !ok {
+		return nil, py.ExceptionNewf(py.TypeError, "str expected (pos 1), not "+args[0].Type().Name)
+	}
+	v, ok := os.LookupEnv(string(k))
+	if ok {
+		return py.String(v), nil
+	}
+	if len(args) == 2 {
+		return args[1], nil
+	}
+	return py.None, nil
+}
+
+// getpid returns the current process id.
+func getpid(self py.Object, args py.Tuple) (py.Object, error) {
+	return py.Int(os.Getpid()), nil
+}
+
+// putenv sets the value of an environment variable named by the key.
+func putenv(self py.Object, args py.Tuple) (py.Object, error) {
+	if len(args) != 2 {
+		return nil, py.ExceptionNewf(py.TypeError, "missing required arguments: 'key:str' and 'value:str'")
+	}
+	k, ok := args[0].(py.String)
+	if !ok {
+		return nil, py.ExceptionNewf(py.TypeError, "str expected (pos 1), not "+args[0].Type().Name)
+	}
+	v, ok := args[1].(py.String)
+	if !ok {
+		return nil, py.ExceptionNewf(py.TypeError, "str expected (pos 2), not "+args[1].Type().Name)
+	}
+	err := os.Setenv(string(k), string(v))
+	if err != nil {
+		return nil, py.ExceptionNewf(py.OSError, "Unable to set enviroment variable")
+	}
+	return py.None, nil
+}
+
+// Unset (delete) the environment variable named key.
+func unsetenv(self py.Object, args py.Tuple) (py.Object, error) {
+	if len(args) != 1 {
+		return nil, py.ExceptionNewf(py.TypeError, "missing one required argument: 'key:str'")
+	}
+	k, ok := args[0].(py.String)
+	if !ok {
+		return nil, py.ExceptionNewf(py.TypeError, "str expected (pos 1), not "+args[0].Type().Name)
+	}
+	err := os.Unsetenv(string(k))
+	if err != nil {
+		return nil, py.ExceptionNewf(py.OSError, "Unable to unset enviroment variable")
+	}
+	return py.None, nil
+}
+
+// os._exit() immediate program termination; unlike sys.exit(), which raises a SystemExit, this function will termninate the program immediately.
+func _exit(self py.Object, args py.Tuple) (py.Object, error) { // can never return
+	if len(args) == 0 {
+		os.Exit(0)
+	}
+	if len(args) == 1 && objectIsInt(args[0]) {
```

Comment:
```suggestion
	arg, ok := args[0].(py.Int)
	if !ok { ...  handle error ... }
```

because:
```py
>>> os._exit("foo")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'str' object cannot be interpreted as an integer
```